doc/ref_arch/kubernetes/chapters/chapter05.md

```diff
@@ -68,19 +68,86 @@ between tenants
 - Enable Role-Based Access Control (RBAC)
 
 ## 5.3 Node Hardening
-Ensure Kubernetes nodes are secure, hardened and configured correctly following
-well known security framework e.g. CIS benchmark, etc. Restrict administrative
-access to Kubernetes nodes while avoiding direct access to nodes for operational
-activities including debugging, troubleshooting, and other tasks.
-
-## 5.4 Authentication & Authorisation
-Secure all connections to a Kubernetes Cluster. Adopt the following security
-authentication mechanisms:
- - Configure user roles and access levels to provide segregation of duties (RBAC)
- - Use multi-factor authentication for all administrative access
- - Use token-based or certificate-based service and session authentication mechanisms
- - Integrated with existing identity management platforms e.g SAML, AD, etc. for
- access control
+### 5.3.1 Node hardening: Securing Kubernetes hosts 		
+Security benchmarking
```

Comment:
@ragashreeshekar Did you men to have "Security benchmarking" as a sub-header? If so, it needs to be before the second paragraph and should be bolded with a blank line before and after.